src/diffusers/models/unet_2d_condition.py

```diff
@@ -109,6 +110,7 @@ def __init__(
         attention_head_dim: Union[int, Tuple[int]] = 8,
         dual_cross_attention: bool = False,
         use_linear_projection: bool = False,
+        num_classes: Optional[int] = None,
```

Comment:
```suggestion
        num_classes_embeds: Optional[int] = None,
```

Reply:
Here "embeds" is important IMO as this defines more or less exactly how many class embedding vectors there are